docs/getting-started/wallets/index.md

```diff
@@ -101,4 +101,64 @@ Strictly speaking, wallet connect is not a wallet; it is an open protocol for co
 ### Bitfy
 
 - [https://bitfy.app/](https://bitfy.app/)
+- Countries: Brazil
 - Platforms: iOS, Android
+
+### Lemon Cash
```

Comment:
Are all of these wallets "Celo compatible" or are some of them "Celo native"? These are all added to the "compatible" section.